src/components/ShoppingCart/ListRow.vue

```diff
@@ -173,6 +184,12 @@ export default {
         ? `${this.$t('cart_purchase_channel')} : ${this.from}`
         : '';
     },
+    tippingAmount() {
+      if (this.productPrice < this.customPrice) {
```

Comment:
Use `Math.max(this.customPrice - this.productPrice, 0)`

Reply:
@AuroraHuang22 I mean we don't need to use `if` when we use math.max

Reply:
[🎨 Handle negative values](https://github.com/likecoin/liker-land/pull/1750/commits/149ccadb556e08ff9b9454b21fe2b6d5eaa42417)

Reply:
[🎨 Simplify tippingAmount calculation](https://github.com/likecoin/liker-land/pull/1750/commits/023549c178d669e40436759ea74a47968db570e1)